### ISSUE TITLE
[Feature Request] Image Gallery component

### ISSUE THREAD

**Issue by YZahringer** [vuetifyjs/vuetify#11177]
### Problem to solve
It would be useful to have a component to manage a gallery of images or videos. It would also support the modal image gallery (lightbox).

### Proposed solution
- https://material.io/components/image-lists
- https://mdbootstrap.com/docs/vue/advanced/gallery/

**Comment by KaelWD** [comment]
This is just the [grid system](https://vuetifyjs.com/en/components/grids/)?

https://vuetifyjs.com/en/components/images/#grid

> lightbox

#1068

> Masonry layout support

#4091

What do you want that isn't either already possible or is covered by another issue?

**Comment by YZahringer** [comment]
Yes, it's a mix of all mentioned issues. Maybe a Gallery component to manage it.

**Comment by billsliu** [comment]
Lightbox gallery will be the most desirable feature, https://mdbootstrap.com/docs/vue/advanced/gallery/

**Comment by danieldanielecki** [comment]
Is it supposed to be implemented anytime soon or we've to rely on `vue-gallery` and other packages?

**Comment by douglasg14b** [comment]
Just the component without content requirements would be great. There are some components right now that assume the content you desire, which makes them unfit for many ancillary use cases, and difficult if not impossible to style & customize.

🤔

**Comment by jongomes** [comment]
+1
